apps/ffdweb-site/public/admin/config.yml

```diff
@@ -146,10 +146,63 @@ collections:
       - *body_config
       - *meta_config_with_title_fallback
 
+  - name: "digest_issues"
+    label: "Digest Issues"
+    folder: "apps/ffdweb-site/src/content/digest-issues/"
+    preview_path: "digest/{{issue-number}}"
+    label_singular: "Digest Issue"
+    create: true
+    extension: "md"
+    slug: "{{issue-number}}"
+    fields:
+      - name: "issue-number"
+        label: "Issue Number"
+        widget: "number"
+        min: 1
```

Comment:
This can be constrained to just ["1", "2"] I believe..  https://decapcms.org/docs/widgets/#select